feed/views.py

```diff
@@ -128,6 +128,12 @@ def like(request):
 	return HttpResponse(response, content_type = "application/json")
 
 
+@login_required

```

Comment:
shouldn't need the method decorator for a helper function, since the calling functions already have the decorator. and in an ideal world (if we had time, which we really don't) the helper function should go in another class/file

Reply:
I was thinking the same thing as it would make more sense to put all this and other similar refactorings in a common class.